src/invite-api.js

```diff
@@ -68,7 +68,7 @@ export class InviteApi extends TypedEmitter {
 
   /**
    * @param {string} peerId
-   * @param {import('./generated/rpc.js').Invite} invite
+   * @param {import('./generated/rpc.js').Invite & { invitorName: string }} invite
```

Comment:
I'm not to sure about this, since it basically means adding `& {invitorName: string}` to a bunch of places and it can become hard to track? If an invite actually contains that field, then maybe we should add it to the proto...
@gmaclennan wdyt??

Reply:
dismiss this. I realize that since we're sending the invite through a protobuf, this needs to be on the `Invite` proto